akka-distributed-data/src/main/resources/reference.conf

```diff
@@ -46,6 +46,50 @@ akka.cluster.distributed-data {
   # after this duration.
   serializer-cache-time-to-live = 10s
   
+  durable {
+    # List of keys that are durable. Prefix matching is supported by using * at the
+    # end of a key.  
+    keys = []
+    
+    # Fully qualified class name of the durable store actor. It must be a subclass
+    # of akka.actor.Actor and handle the protocol defined in 
+    # akka.cluster.ddata.DurableStore. The class must have a constructor with 
+    # com.typesafe.config.Config parameter.
+    store-actor-class = akka.cluster.ddata.MapDbDurableStore
+    
+    use-dispatcher = akka.cluster.distributed-data.durable.pinned-store
+    
+    pinned-store {
+      executor = thread-pool-executor
+      type = PinnedDispatcher
+    }
+    
+    # Config for the MapDbDurableStore
+    mapdb {
```

Comment:
Super fast, lightweight, battle hardened (backs OpenLDAP since years), crash-resistant. Also supports transactions, so you can write multiple kv locations in ACID manner.